build/Targets/Imports.targets

```diff
@@ -360,6 +360,8 @@
   <Target Name="RestoreToolsetCheck" Condition="'$(BuildingProject)' == 'true'">
       <Error Text="Toolset packages have not been restored, run Restore.cmd before building"
              Condition="!Exists('$(ToolsetCompilerPropsFilePath)')" />
+      <Error Text="Analyzer packages have not been restored, run Restore.cmd before building"
```

Comment:
Is this just a test that should really exist on the roslyn-analyzers side of things?